assets/css/main.css

```
@@ -91,6 +91,33 @@ a:hover {
 
 /* FAQ
 ================================ */
+.accordion-button::after {
+  background-image: url(../img/plus-drop-down.svg);
+  }
+
+.accordion-button:not(.collapsed)::after {
+  background-image: url(../img/minus-drop-down.svg);
+}
+
+.accordion-button:focus {
+  border-color: rgba(0,0,0,.125);
+  box-shadow: none;
+}
+
+.accordion-button:not(.collapsed) {
+  color: black;
+  background-color: white;
+  border-bottom: white;
+}
+
+.accordion-flush .accordion-item:last-of-type .accordion-button.collapsed {
+  border-bottom-width: 1px;
+}
+
+.accordion-button {
+  text-align: left;
+  font-family: Arial;
```

Instead of `Arial`, we can use the CSS variable that controls the font for the rest of the document. We should also make the FAQ titles bolder and larger.

```suggestion
  font-family: var(--body-font);
  font-weight: bold;
  font-size: 1.125rem;
```